cmd/nodecmd/create_gcp.go

```diff
@@ -226,6 +227,31 @@ func createGCPInstance(usr *user.User, gcpClient *compute.Service, zone, imageID
 	instanceIDs, elasticIPs, certFilePath, keyPairName, err := createGCEInstances(rootBody, gcpClient, hclFile, zone, imageID, defaultAvalancheCLIPrefix, gcpProjectName, gcpCredentialFilepath)
 	if err != nil {
 		ux.Logger.PrintToUser("Failed to create GCP cloud server")
+		if err.Error() == constants.ErrCreatingGCPNode {
```

Comment:
It seems that we need to print a clear msg that we started to stop all nodes due to create failure

Reply:
can we have the same code for this and for stop command? it seems to be doing pretty similar stuff (stop all nodes of the cluster). this to reduce code/logic repetition